cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -366,7 +374,7 @@ runTxBuildCmd
                      mapM (readFileScriptInAnyLang . unScriptFile) scriptFiles
   txAuxScripts <- hoistEither $ first ShelleyTxCmdAuxScriptsValidationError $ validateTxAuxScripts cEra scripts
   mpparams <- forM mPparams $ \ppFp ->
-    firstExceptT ShelleyTxCmdProtocolParamsError (readProtocolParametersSourceSpec ppFp)
+    firstExceptT ShelleyTxCmdProtocolParamsError (readProtocolParametersSourceSpec' consensusModeParams nid ppFp)
```

Comment:
I have a feeling that `mpparams` is completely ignored here: https://github.com/input-output-hk/cardano-node/blob/master/cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs#L651-L655

Reply:
Thanks for pointing that out.  I see that it's used in the code, despite the printed warning. Let's move the discussion over there: https://github.com/input-output-hk/cardano-node/issues/5052#issuecomment-1505226479